experiment_buddy/experiment_buddy.py

```diff
@@ -232,57 +240,49 @@ def _setup_tb(logdir):
     return tensorboardX.SummaryWriter(logdir=logdir)
 
 
-def _open_ssh_session(hostname: str) -> fabric.Connection:
+def open_ssh_session(hostname: str) -> fabric.Connection:
```

Comment:
not a public method

Reply:
To be moved in a class

Reply:
Revert it here, apply the change to the class commit

Reply:
Reverted